xplique/metrics/fidelity.py

```diff
@@ -625,3 +625,246 @@ def __init__(self,
                  ):  # pylint: disable=R0913
         super().__init__(model, inputs, targets, metric, batch_size,
                          "insertion", baseline_mode, steps, max_percentage_perturbed)
+
+
+class CausalFidelityTab(ExplanationMetric):
```

Comment:
I have the feeling that CausalFidelityTab is just a specific CausalFidelityTS. Do you think it is possible to make it inherits from CausalFidelityTS by setting some of the arguments ? It would save a great deal of lines of codes!

Reply:
CausalFidelityTab exists anymore, InsertionTab and DeletionTab inherits from the class CausalFidelityTS